libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1051,6 +1061,7 @@ - (SFAuthErrorHandlerList *)populateDefaultAuthErrorHandlerList
     self.genericAuthErrorHandler = [[SFAuthErrorHandler alloc]
                                                  initWithName:kSFGenericFailureAuthErrorHandler
                                                  evalBlock:^BOOL(NSError *error, SFOAuthInfo *authInfo) {
+                                                     [[SalesforceSDKManager sharedManager] dismissSnapshot];
```

Comment:
It's a nitpick, but shouldn't this logic technically be in `showRetryAlertForAuthError:alertTag:`?  It's ultimately the presentation of the alert view that necessitates the dismissal of the snapshot view.